index.html

```diff
@@ -3,14 +3,62 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    
-    <title>My first Microverse student project</title>
-
+    <title>Luis's portfolio</title>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
     <link rel="stylesheet" href="css/style.css">
   </head>
-  <body>
 
-    <h1>Hello Microverse!</h1>
+  <body>
+    <header class="header">
+      <nav class="navbar">
+        <a href="#home" class="nav-logo">My Logo</a>
+        <ul class="nav-menu">
+          <li class="nav-item">
+            <a href="#portfolio" class="nav-link">Portfolio</a>
+          </li>
+          <li class="nav-item">
+            <a href="#about" class="nav-link">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="#contact" class="nav-link">Contact</a>
```

Comment:
- [ ] **REQUIRED:** There is no need for these parts on this milestone, in place of it just create a hamburger button that has a `nav` tag.